frmts/zarr/zarrdriver.cpp

```diff
@@ -1443,6 +1573,14 @@ void GDALRegister_Zarr()
         "   <Option name='CACHE_TILE_PRESENCE' type='boolean' "
         "description='Whether to establish an initial listing of present "
         "tiles' default='NO'/>"
+        "   <Option name='MULTIBAND' type='boolean' default='NO' "
+        "description='Whether to expose >= 3D arrays as GDAL multiband "
```

Comment:
Should it be "default='YES'" like in the zarr.rst above?

Reply:
good point . fixed